.github/workflows/premerge.yml

```
@@ -0,0 +1,31 @@
+name: Premerge
+
+on:
+  pull_request:
+    types: [opened, synchronized, reopened]
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v2
+
+      - name: Install Rust toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly
```

```suggestion
          toolchain: nightly-2020-09-27
```

Probably should be this instead